packages/wasm-utils/src/lib.rs

```diff
@@ -1,7 +1,10 @@
 #![allow(dead_code)]
 
+#[cfg(feature = "wee_alloc")]
+#[global_allocator]
+static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
+
 pub mod note;
 pub mod proof;
 pub mod types;
 mod utils;
```

Comment:
We have 2 utils. One in the root, one in the `proof/utils`. None of them are used, so we can delete them

Reply:
Removed the proof/utils.rs